### DISCUSSION TITLE
BoolToEnumConverter

### DISCUSSION THREAD

**Discussion by NebulaSleuth** [xamarin/XamarinCommunityToolkit#1907] [Q&A]
I am trying to use the BoolToEnumConverter, but I keep getting a compile error in my xaml and can't figure out what I am doing wrong.  I keep getting 

"XamlC error XFC0009: No property, BindableProperty, or event found for "TrueValues", or mismatching type between value and property."

My Enum:
`    public enum CheckInStatus
    {
        Unknown,
        BeforeCheckInTime,
        Available,
        AfterCheckInTime,
        CheckedIn
    }`

My Xaml:

    `<?xml version="1.0" encoding="utf-8" ?>
    <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                   xmlns:vm="clr-namespace:LastQueenStanding.ViewModels"
                   xmlns:local="clr-namespace:LastQueenStanding"
                   xmlns:xct="http://xamarin.com/schemas/2020/toolkit"
                   x:Class="LastQueenStanding.Views.CheckInPage"
                   Title="{Binding Title}">
    <ContentPage.Resources>
        <ResourceDictionary>
            <xct:EnumToBoolConverter x:Key="CheckInAvailable">
                <xct:EnumToBoolConverter.TrueValues>
                    <vm:CheckInStatus>Available</vm:CheckInStatus>
                </xct:EnumToBoolConverter.TrueValues>
            </xct:EnumToBoolConverter>
        </ResourceDictionary>
    </ContentPage.Resources>
        <Grid></Grid>  
    </ContentPage>
        
`

**Comment by bijington** [comment]
@NebulaSleuth I believe this is because `TrueValues` expects a list of values and not a single one. I don't recall how to define a list in XAML but you could workaround it by adding your `Available` entry in twice.

```xaml
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
               xmlns:vm="clr-namespace:LastQueenStanding.ViewModels"
               xmlns:local="clr-namespace:LastQueenStanding"
               xmlns:xct="http://xamarin.com/schemas/2020/toolkit"
               x:Class="LastQueenStanding.Views.CheckInPage"
               Title="{Binding Title}">
<ContentPage.Resources>
    <ResourceDictionary>
        <xct:EnumToBoolConverter x:Key="CheckInAvailable">
            <xct:EnumToBoolConverter.TrueValues>
                <vm:CheckInStatus>Available</vm:CheckInStatus>
                <vm:CheckInStatus>Available</vm:CheckInStatus>
            </xct:EnumToBoolConverter.TrueValues>
        </xct:EnumToBoolConverter>
    </ResourceDictionary>
</ContentPage.Resources>
    <Grid></Grid>  
</ContentPage>
```
